packages/core/src/__tests__/unit/application-lifecycle.unit.ts

```diff
@@ -153,6 +198,37 @@ class MyObserver implements LifeCycleObserver {
   }
 }
 
-class FakeComponentWithAnObserver implements Component {
-  lifeCycleObservers = [MyObserver];
+@bind({
+  tags: {
+    [CoreTags.LIFE_CYCLE_OBSERVER]: CoreTags.LIFE_CYCLE_OBSERVER,
+    [CoreTags.LIFE_CYCLE_OBSERVER_GROUP]: 'my-group',
+    namespace: CoreBindings.LIFE_CYCLE_OBSERVERS,
+  },
+  scope: BindingScope.SINGLETON,
+})
+class MyObserverWithBind implements LifeCycleObserver {
+  status = 'not-initialized';
+
+  start() {
+    this.status = 'started';
+  }
+  stop() {
+    this.status = 'stopped';
+  }
+}
+
+@lifeCycleObserver('my-group', BindingScope.SINGLETON)
+class MyObserverWithDecorator implements LifeCycleObserver {
```

Comment:
This class is used from a single test only, please move its definition to that test case. I think it will make the test easier to read.

Reply:
No, it's used in more than one tests.